iceoryx_hoofs/test/moduletests/test_unit_duration.cpp

```diff
@@ -1249,22 +1281,18 @@ TEST(Duration_test, ConvertTimespecWithEpochReferenceFromMaxDurationResultsInSat
     EXPECT_THAT(sut.tv_nsec, Eq(NANOSECONDS));
 }
 
-// END CONVERSION FUNCTION TESTS
-
-// BEGIN CONVERSION OPERATOR TESTS
-
-TEST(Duration_test, OperatorTimevalFromZeroDuration)
+TEST(Duration_test, ConvertTimevalFromZeroDuration)
 {
     ::testing::Test::RecordProperty("TEST_ID", "10d3b209-093c-42c2-b3ab-2f2ac7e53836");
     auto duration = createDuration(0U, 0U);
 
-    struct timeval sut = timeval(duration);
+    struct timeval sut = duration.timeval();
```

Comment:
why not `const auto sut = duration.timeval();`

Applies also to the tests below.

Reply:
I had this in the first place and then decided against it to be sure to receive a `timeval` and not just a struct which coincidentally has member with the same names like a `timeval`.